pkg/config/kube/conversion.go

```diff
@@ -59,9 +70,15 @@ func ConvertProtocol(name string, proto coreV1.Protocol) protocol.Instance {
 			name = name[:i]
 		}
 		p := protocol.Parse(name)
-		if p != protocol.UDP && p != protocol.Unsupported {
+		if p != protocol.UDP {
 			out = p
 		}
 	}
+
+	// Make TCP as default protocol for well know ports if protocol is not specified.
+	if _, has := wellKnownPorts[port]; has && out == protocol.Unsupported {
```

Comment:
nit: add a utility function `isWellKnownPort(port)` ... this will clean up this statement.